config/authorization/config.ex

```diff
@@ -307,6 +308,12 @@ defmodule Acl.UserGroups.Config do
               resource_types: email_resource_types()
             }
           },
+          %GraphSpec{
+            graph: "http://mu.semte.ch/graphs/staatsblad",
+            constraint: %ResourceConstraint{
+              resource_types: publication_resource_types()
```

Comment:
Zijn dit niet te veel resource types? Willen we niet enkel `eli:LegalResource` in deze graph?
Met deze config lijkt het mij dat bij het aanmaken van een nieuwe `pub:Publicatieaangelegenheid` die zowel in de kanselarij-graph als de staatsblad-graph zal geinsert worden.

Reply:
Het lijkt wel erg veel om over het hoofd gezien te hebben, maar dat moet toch zijn wat er gebeurd is. Dank voor het opmerken @ValenberghsSven & @erikap . Ik heb aangepast om wat aan backend-configuratie reeds voorzien was, te hergebruiken.